examples/ecr/respositorypolicy-json.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: ecr.aws.crossplane.io/v1alpha1
+kind: RepositoryPolicy
+metadata:
+  name: example
+  labels:
+    region: us-east-1
+spec:
+  forProvider:
+    region: us-east-1
+    repositoryName: example
+    RawPolicy:  |
```

```suggestion
    rawPolicy:  |
```

Just skimmed the PR and noticed this, I have not reviewed it :)

---

doh, nice catch.